Formula/y/yazi.rb

```diff
@@ -30,9 +30,4 @@ def install
     ENV["VERGEN_GIT_SHA"] = tap.user
     system "cargo", "install", *std_cargo_args(path: "yazi-fm")
   end
-
-  test do
-    # yazi is a GUI application
-    assert_match "yazi #{version}", shell_output("#{bin}/yazi --version").strip
-  end
```

Comment:
we need test

Reply:
Any idea how to solve this issue with a test?

Reply:
@chenrui333 pls 👆🏽